### ISSUE TITLE
How to implement 'Auto Pause' Feature

### ISSUE THREAD

**Issue by gooridevietnam** [transistorsoft/flutter_background_geolocation#1190]
## Your Environment
* Plugin version:
* Platform: iOS or Android
* OS version:
* Device manufacturer / model:
* Flutter info (`flutter doctor`):
* Plugin config:
```dart <-- Syntax highlighting: DO NOT REMOVE -->
PASTE_YOUR_CODE_HERE
```

## Expected Behavior
Sorry I am new to your package. Just research recently.
I am planning develop a feature 'Auto Pause'. My expectation:
1. When user is not moving about 10s, auto detect -> pause tracking.
2. When use moves(for example 10m over), it will auto detect and start tracking again.

I can see onMotionChange event. I tried to use your example app. But seems like it doesn't work like my expectation above. Is there any support in your package?

Sorry if I have misunderstood something .

## Steps to Reproduce

1.
2.
3.
4.

## Debug logs

<details>
	<summary>Logs</summary>

``` <!-- syntax-highligting:  DO NOT REMOVE -->
PASTE_YOUR_LOGS_HERE
```

</details>

**Comment by christocracy** [comment]
Read the Wiki "Philosophy of Operation".

The lowest possible distance to detect the device is moving typically requires movement of about 200 meters.

The Android plugin is allowed to use the motion api to automatically start/stop tracking.  See API docs Config.stopTimeout.  You *could* try using `stopTimeout: 0` for Android and the plugin will stop tracking as soon as the motion API reports `still`. 

On iOS, the best you'll ever get is at least 200 meters before tracking resumes.

>When user is not moving about 10s, auto detect -> pause tracking.
>When use moves(for example 10m over), 

You'll never be able to achieve that with **exact** times

**Comment by gooridevietnam** [comment]
Thank you. One more question.
How long does it takes to become to "Stationary" State. Can I reduce that time to something likes 10 seconds

**Comment by christocracy** [comment]
See API docs Config.stopTimeout (minutes)

**Comment by gooridevietnam** [comment]
I am using your application on CH play for testing. I changed stopTimeout to 0. After that, I enabled plugin and clicked ChangePace to force change to moving. But actually I do not move anywhere. From my understanding, it should be changed to "Stationary" State after a specific time. But it is not, so I want to ask how long does it takes to detect a "Stationary" State, and can I reduce that time?

I read Philosophy of Operation, but seems there is no mention about this information.
